#!/usr/bin/python3

def raise_exception_msg(message=""):
    """Raise a NameError exception by giving out  a message."""
    raise NameError(message)
